app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -117,6 +118,12 @@
 
     public static final String EXTRA_FILES = "commons_image_exta";
 
+    /**
+     * Stores all nearby places found and related users response for
+     * each place while uploading media
+     */
+    public static HashMap<Place,Boolean> existingPlaces;
```

Comment:
I think we need a more explicit name for this.
How about `nearbyPopupAnswers`?
Or any other idea?

Reply:
@nicolas-raoul Yes, `nearbyPopupAnswers` will be better. One other name can be `nearbyPopupResponses`. Tell me which is more accurate to you I will change it.

Thanks

Reply:
You are right, I thought about `Responses` first, but we'd better avoid possible ambiguity with _HTTP responses_, I think.